core/src/components/action-sheet/action-sheet.ios.vars.scss

```diff
@@ -97,19 +97,19 @@ $action-sheet-ios-sub-title-padding-start:                        $action-sheet-
 $action-sheet-ios-button-height:                                  56px !default;
 
 /// @prop - Padding of the action sheet button
-$action-sheet-ios-button-padding:                                 18px !default;
+$action-sheet-ios-button-padding:                                 14px !default;
```

Comment:
Now that we are using `min-height` instead of `height`, the default height was being set to `64px` due to this padding. Our old `height: 56px` masked the problem. You can see it in the following screenshot where the icon extends beyond the boundary of the button height (excluding the padding)
![IMG_0008](https://github.com/ionic-team/ionic-framework/assets/2721089/201c831d-491e-4bd8-bc9c-d66f06b2f5ca)